### PR TITLE
Implement tile generation without creating temp tables

### PR DESCRIPTION
**Pull request by killev**
I have suspicions that tile generation leads to outages because it's transactional and can't execute SQL queries in parallel. So I avoid transactions, rewrite SQL so that is uses singe query, avoid using TEMP TABLES. As a result, the SQL query runs 5 time longer (0.025sec before vs 0.15s after, but seems it improves performance in general)

### CODE REVIEW

**Review by roman-stolar — DISMISSED** [review]
Sync up on call with Peter, Approved.

**Review by roman-stolar — APPROVED** [review]
LGTM

**Review by mazursasha1990 — DISMISSED** [review]
Approved

**Review by mazursasha1990 — APPROVED** [review]
Approved

File: doc/release/RELEASE-NOTES.md [hunk 1]
```diff
@@ -23,9 +23,9 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 * [OSDEV-1069](https://opensupplyhub.atlassian.net/browse/OSDEV-1069) - The following changes have been made:
     * Changed the Postgres Docker image for the database to use the official one and make the local database setup platform-agnostic, so it doesn't depend on the processor architecture.
     * Built the PostGIS program from source and installed it to avoid LLVM-related errors inside the database Docker container during local development.
+* Make tile generation endpoint transaction-less and remove `CREATE TEMP TABLE` statement.
+* Added command `reindex_database`.
 * [OSDEV-1089](https://opensupplyhub.atlassian.net/browse/OSDEV-1089) Change format gunicurn logs not pass IP address to AWS CloudWatch.
-* Added command `reindex_database`
```
Comment (roman-stolar) [inline doc/release/RELEASE-NOTES.md:27]: Should we remove it from Release-Notes?

**Review by mazursasha1990 — APPROVED** [review]
Approved

File: doc/release/RELEASE-NOTES.md (hunk 1 above)
Comment (killev) [reply doc/release/RELEASE-NOTES.md:27]: Why?

Comment (killev) [reply doc/release/RELEASE-NOTES.md:27]: It came from another PR and I fixed conflict. So we need it I think

**Review by roman-stolar — APPROVED** [review]
LGTM

**Pull request merged**